csharp/src/Ice/Proxy.cs

```diff
@@ -651,12 +575,12 @@ internal class InvokeTaskCompletionCallback : TaskCompletionCallback<Object_Ice_
             }
             if (done)
             {
-                SetResult(new Object_Ice_invokeResult(true, null));
+                SetResult(new IncomingResponseFrame(ReplyStatus.OK, og.GetIs()));
```

Comment:
should we use a cached Empty frame here, seems this is only call for oneway invocations that are done after sent

Reply:
This code needs to be refactored. Currently GetIs is always initialized so we're not wasting anything:
```Is = iss ?? new Ice.InputStream(communicator, Ice.Util.CurrentProtocolEncoding);```